doc/python/README.txt

```diff
@@ -0,0 +1,37 @@
+INTRODUCTION
+============
+
+This folder includes a python script which runs the C emulators for the ASRC, SSRC, OS3 and DS3 modules.
+For each model we present test signals - both a single tone to test the THD performance and a range of tones to
+observe roll-off at the top of the band.
+Using an un-windowed FFT we calculate the SNR.

```

Comment:
Please avoid using personal pronouns in customer-visible documentation.

I have seen some use of 'we' in source file comments as well.  That's only a problem if they will be included in user-visible documentation via doxygen.

Reply:
I have updated this - I will not name the author.